src/Bridge/Symfony/Bundle/Resources/config/security.xml

```diff
@@ -6,7 +6,7 @@
 
     <services>
         <service id="api_platform.security.resource_access_checker" class="ApiPlatform\Core\Security\ResourceAccessChecker" public="false">
-            <argument type="service" id="api_platform.security.expression_language" on-invalid="null" />
+            <argument type="service" id="security.expression_language" on-invalid="null" />
```

Comment:
Can you create an alias `api_platform.security.expression_language` please? It will be useful for people wanting to inject a custom service.